_common/core_ref.md

```diff
@@ -1,5 +1,21 @@
 ---
-layout: externalurl
-redirect_url: https://core.blockstack.org/
+layout: core
 title: "Stacks Node API"
+description: Interacting with the Blockstack Blockchain
+permalink: /:collection/:path.html
 ---
+
+With the launch of Stacks 2.0, a new version of the Blockstack blockchain was released. You currently have two ways of interacting with the blockchain:
```

Comment:
I sometimes use [Hemmingway](http://www.hemingwayapp.com/) to point this out, but usually better to write in the active voice: `There are two ways of interacting with the blockchain`.